standards/decommissioning.md

```diff
@@ -1,23 +1,42 @@
 # Decommissioning and Deprecating Apps
 
-NYPL creates different applications for user interfaces, data pipeline flow, APIs, and more. While we maintain these applications, sometimes it's better to move from one platform to another for many reasons.
+NYPL maintains various applications for user interfaces, data pipeline flow, APIs, and more. In order to align with product and department strategy and needs, it's possible that applications need to be decommissioned. This can happen for various reasons:
```

Comment:
```suggestion
NYPL maintains various applications for user interfaces, data pipeline flow, APIs, and more. To align with product and department strategy and needs, it's possible that applications need to be decommissioned. This can happen for various reasons:
```